react_native_fundamentals/presentation.html

```diff
@@ -2,43 +2,1137 @@
+          </aside>
+        </section>
+
+        <section class="several-pictures-2-slide-with-description">
+          <h4 class="slide-name">Flex Wrap</h4>
+          <div>
+            <div class="fragment" data-fragment-index="1">
+              <img src="assets/images/flexwrap_wrap.png" alt="simple" />
+              <p>wrap</p>
+            </div>
+            <div class="fragment" data-fragment-index="2">
+              <img src="assets/images/flexwrap_nowrap.png" alt="simple" />
+              <p>nowrap</p>
+            </div>
+          </div>
+          <aside class="notes">
+            The flexWrap property is set on containers and it controls what happens when children overflow the size of the container along the main axis. By default, children are forced into a single line (which can shrink elements). If wrapping is allowed, items are wrapped into multiple lines along the main axis if needed.
+When wrapping lines, alignContent can be used to specify how the lines are placed in the container.
+          </aside>
+        </section>
+
+        <section class="text-and-conding-slide">
+          <h4 class="slide-name">Platform Specific Code</h4>
+          <div class="center-align">
+            <p>Platform.OS</p>
+            <pre data-id="code-animation">
+						<code class="js" data-trim data-line-numbers>
+              if(Platform.OS === 'ios') {}           
+						</code>
+					  </pre>
+            <p>Platform.Version</p>
+            <pre data-id="code-animation">
+						<code class="js" data-trim data-line-numbers>
+              if(Platform.Version === 25) {}   
+						</code>
+					  </pre>
+            <p>Platform.select</p>
+            <pre data-id="code-animation">
+						<code class="js" data-trim data-line-numbers>
+              const Component = Platform.select({
+                ios: () => require('ComponentIOS'),
+                android: () => require('ComponentAndroid'),
+              });
+						</code>
+					  </pre>
+          </div>
+          <aside class="notes">
+            Sometimes you may want to implement separate visual components for Android and iOS.
+* The Platform API allows us to conditionally apply different styles or properties in our component
+based on the device’s operating system.
+*  The OS attribute of the object returns either iOS or android depending on the user’s device. 
+•There is also a Platform.select method available, that given an object where keys can be one of 'ios' | 'android' | 'native' | 'default', returns the most fitting value for the platform you are currently running on.
+•The Platform module can also be used to detect the version of the Android Platform or system version of iOS
+•Platform-specific extensions
+•When your platform-specific code is more complex, you should consider splitting the code out into separate files. React Native will detect when a file has a .ios. or .android. extension and load the relevant platform file when required from other components.
```

Comment:
> React Native will detect when a file has a .ios. or .android. extension 

can you show it on a slide please, because it seems to me that without example it will be difficult to understand what it is about

Reply:
I added a separate slide about this with an example